src/frontend/mod.rs

```diff
@@ -411,6 +411,7 @@ impl<'a> Frontend<'a> {
                 } else {
                     Some(WithRange(language, language_range))
                 },
+                attributes: cskvp.take_all_double(),
```

Comment:
Maybe we could take some inspiration from html, and have a `style` attribute instead that contains options that should be forwarded? I'm not completely sure either.